### DISCUSSION TITLE
The difference between 501(c)3 and 501(c)6 organizations

### DISCUSSION THREAD

**Discussion by claredillon** [InnerSourceCommons/foundation-governance#6] [General]
The InnerSource Commons Foundation (ISC) is incorporated as a 501(c)(3) non-profit organization. 

Some other organizations in the ISC ecosystem are 501(c)(6) organizations (e.g. Linux Foundation). A question was raised in a discussion about our mission and charter about the differences. 

The following information is summarized from a blog post on the topic from donorbox.org. [Check it out](https://donorbox.org/nonprofit-blog/501c3-vs-501c6#:~:text=The%20primary%20difference%20between%20the,non%2Dcharitable%20organizations%20as%20well.) for more details. 

The primary difference between a 501(c)(3) and 501(c)(6) is in terms of the purpose of the organization. 501(c)(3) organizations must serve the public. 501(c)(6) organizations are formed to serve their members. The purpose of the ISC is limited by the IRS (US Internal Revenue Service) to being an educational and scientific institution. 

This designation comes with benefits (e.g. tax exemptions for donors), as well as specific accountabilities (e.g. reporting accountabilities) and constraints in how we operate (e.g. cannot take part in political campaigns).

## Purpose and activities

501(c)(3) organizations’ primary goal is charitable or educational. They must perform activities that benefit the public through services like housing, food distribution, [education](https://donorbox.org/university-fundraising), religion, or other publicly beneficial actions.

In the case of the IS, our mission is educational and scientific.

Here is a snapshot of our mission as registered with the IRS in 
<img width="533" alt="image" src="https://user-images.githubusercontent.com/32961070/213858233-6917cfc2-e61c-40eb-8632-1cd7447f3009.png">

It reads "to support and maintain an organization of individuals with a common interest in open source best practices for software development within proprietary organization ("InnerSource"), promoting the discussion of and stimulating knowledge concerning the understanding, adoption and practice of InnerSource and the making of distributions to organizations that qualify as exempt organizations under 501(c)(3) of the Internal Revenue Code, or the corresponding section of any future federal tax code. 

In addition to providing services, 501(c)(3) organizations are responsible for informing and educating the public about their [missions](https://donorbox.org/nonprofit-blog/mission-statement-examples) and causes.

501(c)(3) nonprofits can advocate and lobby for their causes within reason.

The IRS does not see 501(c)(6) organizations as charitable organizations. Instead, these organizations are formed to support their members’ business activities.

501(c)(6) organizations have less strict regulations regarding lobbying.

## Summary of differences
![image](https://user-images.githubusercontent.com/32961070/213855999-75d99b38-ad73-49f6-acee-da4759b201a3.png)

**Comment by rrrutledge** [comment]
This is interesting.  What started this conversation?

**Reply by claredillon** [reply]
We were discussing how the ISC mission is evolving and wanted to double check the related obligations.
